gui/wxpython/wxplot/scatter.py

```diff
@@ -318,19 +318,63 @@ def _toolbarData(self):
         """Toolbar data"""
         return self._getToolbarData(
             (
-                ("addraster", BaseIcons["addRast"], self.parent.OnSelectRaster),
+                (
+                    ("addraster", _("Add raster")),
+                    BaseIcons["addRast"],
+                    self.parent.OnSelectRaster,
+                ),
                 (None,),
-                ("draw", PlotIcons["draw"], self.parent.OnCreateScatter),
-                ("erase", BaseIcons["erase"], self.parent.OnErase),
-                ("drag", BaseIcons["pan"], self.parent.OnDrag),
-                ("zoom", BaseIcons["zoomIn"], self.parent.OnZoom),
-                ("unzoom", BaseIcons["zoomBack"], self.parent.OnRedraw),
+                (
+                    ("draw", _("Draw/re-draw plot")),
+                    PlotIcons["draw"],
+                    self.parent.OnCreateScatter,
+                ),
+                (
+                    ("erase", _("Erase display")),
+                    BaseIcons["erase"],
+                    self.parent.OnErase,
+                ),
+                (
+                    ("drag", _("Pan")),
+                    BaseIcons["pan"],
+                    self.parent.OnDrag,
+                ),
+                (
+                    ("zoom", _("Zoom in")),
+                    BaseIcons["zoomIn"],
+                    self.parent.OnZoom,
+                ),
+                (
+                    ("unzoom", _("Zoom back")),
```

Comment:
Return to previous zoom (just for consistency)

Reply:
Yes I agree.